cfg/FTCPlanner.cfg

```diff
@@ -25,12 +25,18 @@ grp_pid.add("kp_ang", double_t, 0, "KP for angle controller", 1.0, 0.0, 50.0)
 grp_pid.add("ki_ang", double_t, 0, "KI for angle controller", 0.0, 0.0, 5.0)
 grp_pid.add("ki_ang_max", double_t, 0, "KI windup for speed controller angle", 10.0, 0.0, 500.0)
 grp_pid.add("kd_ang", double_t, 0, "KD for angle controller", 0.0, 0.0, 5.0)
+grp_pid.add("kp_ang_rotate", double_t, 0, "KP for angle in pre/post rotate", 1.0, 0.0, 50.0)
+grp_pid.add("ki_ang_rotate", double_t, 0, "KI for angle in pre/post rotate", 0.0, 0.0, 5.0)
+grp_pid.add("ki_ang_max_rotate", double_t, 0, "KI windup for angle in pre/post rotate", 10.0, 0.0, 500.0)
+grp_pid.add("kd_ang_rotate", double_t, 0, "KD for angle in pre/post rotate", 0.0, 0.0, 5.0)
```

Comment:
Is it intended that these values are the same as those without `_rotate`? So different settings would be *possible*, but not used by default?

Reply:
Exactly, didn't want to change current behaviour but I'm using higher values

Reply:
Which values do you use then? I'll see if the target vs. error helps, but if not, it would be good to know which values to increase how much.

Reply:
I'm using completely different values as I've change how the motors are controlled.
Looking at https://github.com/ClemensElflein/open_mower_ros/blob/main/src/open_mower/params/ftc_local_planner.yaml
I would start with kp_ang_rotate: 3.5, ki_ang_rotate: 0.5